Assets/Scripts/Managers/BackgroundMusicManager.cs

```diff
@@ -0,0 +1,81 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class BackgroundMusicManager : SingletonBehaviour<BackgroundMusicManager>
+{
+	[Tooltip( "Seconds it takes to fade out a song" )]
+	[SerializeField] float fadeOutTime = 0f;
+	[SerializeField] float fadeInTime = 0f;
```

Comment:
Is there a reason we'd want to have a different amount of time for fading a song in and out? If one's fading out and another is fading in shouldn't that happen in the same amount of time?